SF iOS/SF iOS/EventDetails/EventDetailsViewController.m

```diff
@@ -83,10 +83,15 @@ - (void)viewDidLoad {
 
     self.travelTimesView = [[TravelTimesView alloc]
                             initWithDirectionsRequestHandler:^(TransportType transportType) {
-        [DirectionsRequest requestDirectionsToLocation:self.event.location.location
-                                              withName:self.event.venueName
-                                    usingTransportType:transportType];
-    }];
+                                [DirectionsRequest requestDirectionsToLocation:self.event.location.location
+                                                                      withName:self.event.venueName
+                                                            usingTransportType:transportType
+                                                                    completion:^(BOOL success) {
+                                                                        if (!success) {
+                                                                            // show error
+                                                                        }
+                                                                    }];
```

Comment:
We briefly discussed this but why is this necessary?

Reply:
its mostly for cases like where someone an app that they don't have installed, e.g.: they ask to use Google Maps but never got around to installing the app / deleted it after changing the setting

Reply:
will update it in a follow-up PR, this one was getting big enough as it is…

Reply:
Right, just hit the spot where I don't have google installed. Is there a way to pre-check that they have it and not even allow the swipe experience?

Reply:
I thought I had said this already, but I ran into the issue where I didn't have Google Maps installed. Is there a way for us to detect specific app installation state these days with iOS? I thought you could like pre-register that or something. I can dig around.

Reply:
hashtag GitHub UI messed me up.